leftwm-core/src/layouts/layout_mode.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+/// Leftwm now has 2 layout modes, Workspace and Tag. These determine how layouts are remembered.
```

Comment:
```suggestion
/// Leftwm has 2 layout modes, Workspace and Tag. These determine how layouts are remembered.
```
This temporal reference makes no sense in doc comments, right?